### PR TITLE
FeedItem container

### PR DESCRIPTION
**Pull request by thinktwice13**
screenshot from 2018-08-24 17-53-27

**Requirements**
```
FeedItemContainer.propTypes = {
  component: PropTypes.func.isRequired,
  item: PropTypes.object.isRequired
}
```

**Usage**
``` 
 <FeedItem
    component={COMPONENT}
    item={FEED_ITEM_DATA},/>
```
**TODO** Check titles
closes to #74

### CODE REVIEW

File: src/components/FeedPortal/components/FeedItem.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from "react"
+import PropTypes from "prop-types"
+import timeSince from "../../utilities/timeSince"
+import "./FeedItem.css"
+
+const FeedItemContainer = ({
+  component: Component,
+  feed,
```
Comment (the-vampiire) [inline src/components/FeedPortal/components/FeedItem.jsx]: item

File: src/components/FeedPortal/components/FeedItem.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  timestamp,
```
Comment (the-vampiire) [inline src/components/FeedPortal/components/FeedItem.jsx]: timetamp, title, avatar from `item` prop

File: src/components/FeedPortal/components/FeedItem.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        </div>
+      </div>
+      <Component {...feed} {...props} />
+    </Fragment>
+  )
+}
+
+FeedItemContainer.propTypes = {
+  component: PropTypes.func.isRequired,
+  feed: PropTypes.object.isRequired,
```
Comment (the-vampiire) [inline src/components/FeedPortal/components/FeedItem.jsx]: `item`

**Comment by serpient** [comment]
```

let newsFeedData = {
  "newsfeed": {
    "id": "TVRGMmIzbGhaMlU9YWxs",
    "items": [
      {
        "id": "MTF2b3lhZ2U=",
        "type": "NewsfeedVoyage",
        "timestamp": 1535090610661,
        "user": {
          "id": "6",
          "username": "the-vampiire",
          "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
        },
        "voyage": {
          "id": "11",
          "title": "Halloween Voyage"
        }
      },
      {
        "id": "MXN0YW5kdXA=",
        "type": "NewsfeedStandup",
        "timestamp": 1535088249192,
        "user": {
          "id": "6",
          "username": "the-vampiire",
          "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
        },
        "standup": {
          "id": "1",
          "is_expired": false,
          "progress_sentiment": "green",
          "expiration": 1535692113990
        }
      },
      {
        "id": "MTF2b3lhZ2U=",
        "type": "GithubActivityIssue",
        "timestamp": 1535090610661,
        "user": {
          "id": "6",
          "username": "the-vampiire",
          "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
        },
        "repo": {
          "id": "MDEwOlJlcG9zaXRvcnkxNDQ0NTQ1MDQ=",
          "url": "https://github.com/the-vampiire/dynamic-forms",
          "repo_name": "the-vampiire/dynamic-forms",
        },
        "issue": {
          "user": {
            "username": "the-vampiire",
            "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
          },
          "status": "OPEN",
          "title": "Refactor/dynamic form structure",
          "url": "https://github.com/the-vampiire/dynamic-forms/pull/3",
          "timestamp": "2018-08-21T21:19:21Z"
        }
      },
      {
        "id": "MTF2b3lhZ2U=",
        "type": "GithubActivityPullRequest",
        "timestamp": 1535090610661,
        "user": {
          "id": "6",
          "username": "the-vampiire",
          "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
        },
        "repo": {
          "id": "MDEwOlJlcG9zaXRvcnkxNDQ0NTQ1MDQ=",
          "url": "https://github.com/the-vampiire/dynamic-forms",
          "repo_name": "the-vampiire/dynamic-forms",
        },
        "pull_requests": {
          "user": {
            "username": "the-vampiire",
            "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
          },
          "status": "OPEN",
          "title": "Refactor/dynamic form structure",
          "url": "https://github.com/the-vampiire/dynamic-forms/pull/3",
          "timestamp": "2018-08-21T21:19:21Z",
          "files_changed": 11
        }
      }
    ]
  }
}
```

**Pull request merged**